frontend/views/ConfigView.tsx

```diff
@@ -34,7 +32,7 @@ const Controls = styled.div`
 `;
 
 interface ConfigViewProps {
-	store: Store<AppStore, AppActions>;
+	buttonAction: (allVisibile: boolean) => void;
```

Comment:
`buttonAction` is not the greatest name. Think of it as an API, `Route` is exposing an API to `View`. Now you go to the documentation and see an endpoint `buttonAction`. I can only guess what you'd say about the author of that API :D.

Go with something more explicit. I won't give any suggestions so you can ponder it on your own.